pkg/eval/testutils_unix.go

```diff
@@ -0,0 +1,14 @@
+// +build !windows,!plan9,!js
+
+package eval
+
+import (
+	"syscall"
+)
+
+func ExitWaitStatus(exit uint32) syscall.WaitStatus {
+	// The exit<<8 is gross but I can't find any exported symbols that would
+	// allow us to construct WaitStatus. So assume legacy UNIX encoding
```

Comment:
> legacy UNIX encoding

AFAIK there isn't a "modern UNIX" encoding.

Reply:
True but in this instance I'm using this meaning: a thing handed down by a predecessor. The intent is to make it clear we're relying on behavior established long ago but is not guaranteed to be true if we run on a UNIX-like OS.